apollo-router/src/plugins/telemetry/consts.rs

```diff
@@ -18,8 +18,9 @@ pub(crate) const REQUEST_SPAN_NAME: &str = "request";
 pub(crate) const QUERY_PLANNING_SPAN_NAME: &str = "query_planning";
 pub(crate) const HTTP_REQUEST_SPAN_NAME: &str = "http_request";
 pub(crate) const SUBGRAPH_REQUEST_SPAN_NAME: &str = "subgraph_request";
+pub(crate) const QUERY_PARSING_SPAN_NAME: &str = "parse_query";
```

Comment:
This looks like a "drive-by" fix. Should this be called out in the changelog as additional functionality or split into its own PR?

Reply:
I guess it was https://github.com/apollographql/router/pull/5609/files#diff-cab56c50a115affb2a92de4523cc0b85f6c6b7f921693f8e7e5b4c3b34eb78e2L35 originally?  (in another file)

Reply:
Yeah, I just moved this, it's not a rename or anything. It's not changelog worthy.